### DISCUSSION TITLE
[NMS] Not getting metrics on master domain for bare metal

### DISCUSSION THREAD

**Discussion by ShubhamTatvamasi** [magma/magma#5322] [Q&A] [answered]
I am unable to see metrics on my master, I think this is the prometheus issue. Can somebody help me out?

![Screenshot from 2021-03-05 11-25-08](https://user-images.githubusercontent.com/14152150/110073683-cb5ede00-7da5-11eb-8613-0d4c2f4c45c4.png)

**Comment by karthiksubraveti** [comment, marked as answer]
I have seen this problem as well locally. Do you see any organization specific metrics when you log into a NMS organization ?

**Reply by ShubhamTatvamasi** [reply]
Yes, for `magma-test` organization metrics are showing.

**Reply by karthiksubraveti** [reply]
Oh ok. This could be a bug cause the associated metric no longer exist. Can you open an issue for this and add NMS label to be associated with it. If you are interested in fixing this issue, I can provide some code pointers which can help. Let me know.

**Reply by ShubhamTatvamasi** [reply]
@karthiksubraveti I have created an issue for this, could you please assign this to me. https://github.com/magma/magma/issues/5348
